src/randoop/experiments/Execute.java

```diff
@@ -32,9 +32,13 @@ public static void main(String[] args) {
       throw new Error(e);
     }
     for (String className : covClassNames) {
-      Class<?> cls = Reflection.classForName(className);
-      System.out.println(cls.toString() + " " + Coverage.getBranches(cls).size());
-      covClasses.add(cls);
+      try {
```

Comment:
Can you make the try clause cover fewer lines?  It doesn't need to cover the `covclasses.add(cls);" line, and I don't think it is necessary for the `getBranches` line either.  More generally, please make sure all try clauses (or at least the ones you added) have minimal scope.